src/main/java/pl/coderstrust/accounting/logic/pdf/CreatingTable.java

```diff
@@ -0,0 +1,71 @@
+package pl.coderstrust.accounting.logic.pdf;
+
+
+import org.apache.pdfbox.pdmodel.common.PDRectangle;
+import org.apache.pdfbox.pdmodel.font.PDFont;
+
+import java.util.List;
+
+public class CreatingTable {
+
+  private Table table = new Table();
+
+  public CreatingTable setHeight(float height) {
+    table.setHeight(height);
+    return this;
+  }
+
+  public CreatingTable setNumberOfRows(Integer numberOfRows) {
```

Comment:
isn't int enough? it's preferred over Integer when null value is not needed

Reply:
done